.github/workflows/build-and-test.yml

```diff
@@ -39,6 +39,7 @@ jobs:
         uses: dtinth/setup-github-actions-caching-for-turbo@v1
 
       - run: "yarn install --frozen-lockfile"
+      - run: "yarn check-references"
```

Comment:
it may be worth adding documentation to the error message that this command may produce explaining what the error is and that there is a convenient one-liner that would fix the error. I suspect that would save people some time in the future.

Reply:
Done in new PR